src/scripts/components/colour/Time.js

```diff
@@ -5,10 +5,12 @@ import { TimeHelper } from '../../modules/timehelper';
 export const Time = (props) => {
   const time = props.time;
 
-  var hour = time.hour;
+  var hour = parseInt(time.hour);
```

Comment:
It's necessary for

```js
hour = TimeHelper.pad(hour === 0 ? 12 : hour);
```

Before this was _only_ running after subtracting 12 (which implicitly converts `hour` to an integer).

There are a few alternatives though:

1. Subtract 0 from `hour` when `!time.pm`

2. Use `==` instead of `===`